Model/CustomItemExportSchedulerModel.php

```diff
@@ -199,7 +205,7 @@ private function addCustomItemsToCsvFile(array $customFields, CustomObject $cust
 
                     $rowData   = $savedRow;
                     $rowData[] = implode(',', $results);
-                    $this->exportHelper->echoTouchJob();
+                    $this->eventDispatcher->dispatch(CustomItemEvents::ON_PROCESSING_FILE);
```

Comment:
I think as this is used in allydebundle, we should add this check as well
`if ($this->eventDispatcher->hasListeners(CustomItemEvents::ON_PROCESSING_FILE)) {`

Reply:
thats a good point.